doc/releases/release-notes-3.1.rst

```diff
@@ -143,6 +143,8 @@ Bluetooth
     function has been deprecated.  A :c:func:`subscribe` callback
     function has been added to replace it.
 
+  * :c:func:`bt_disable` was added to enable caller to disable bluetooth stack.
+
```

Comment:
Did you mean to put this in this commit?

Reply:
Right, split into a dedicated commit.